### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by adamhstewart**

### CODE REVIEW

**Comment by IshaDakota** [comment]
@amitaibu can you give me a hand by pointing out where the app sets the x- y- limits on the position of the magnets?

**Comment by IshaDakota** [comment]
^^ never mind...

the_team

no results
the_team

**Comment by adamhstewart** [comment]
fireshot capture 534 - the team - http___localhost_3000_team_


image

**Comment by adamhstewart** [comment]
@nedSaf would you mind reviewing this page of the redesign as it touches the elm app quite a bit.

File: team_app/src/App/Update.elm [hunk 1]
```diff
@@ -40,10 +40,10 @@ init flags =
 
                         -- Adjusting the random numbers.
                         x =
-                            Basics.floor <| toFloat randomNumber1 * 1.19
+                            Basics.floor <| toFloat randomNumber1 * 1.19 +25
```
Comment (nedSaf) [inline team_app/src/App/Update.elm]: Why do we add 25 here? if to increase the number, can't we just increase `1.19`?

Comment (adamhstewart) [reply team_app/src/App/Update.elm]: I don't know - basic math is not my strength. But wouldn't increasing the multiplier just increase the spread? I wanted a shift.

Comment (nedSaf) [reply team_app/src/App/Update.elm]: So the purpose is moving the whole thing a bit down?

Comment (amitaibu) [reply team_app/src/App/Update.elm]: @bitamar is the brain behind those majestic magnets - so he might have a clue to what are the numbers. Obviously I don't 😉

Comment (bitamar) [reply team_app/src/App/Update.elm]: It was too long ago, so I don't remember what those numbers mean. But I've been meaning to improve the way they're responding to being pushed, so I'll have a look later.

Comment (adamhstewart) [reply team_app/src/App/Update.elm]: It turns out that it was a markup issue - lot's of `div` containers and I [needed to get the map/grid tabs out of the one containing the magnets](https://github.com/Gizra/gizra.com/commit/9807bf0178b66cfbfd2de387f5bc939e46655dbc).

@bitamar if you have some improvements, have at it.

**Pull request merged**